src/javascript/button.js

```diff
@@ -22,7 +22,9 @@ var uuid = require('node-uuid');
  */
 function InputButton(command, opts) {
   opts = opts || {};
-  opts.insightsVerb = opts.insightsVerb || 'PostButtonClick';
+  opts.insightsVerb = opts.insightsVerb === null
```

Comment:
Seems like you shouldn't need to do this at all. Shouldn't `Button.call(this, command, opts);` take care of it?